### ISSUE TITLE
Release to pipy

### ISSUE THREAD

**Issue by muschellij2** [onnela-lab/forest#244]
Is there some guidance on when versions will be updated?  Currently, the recommendation is to use:
```
pip install git+https://github.com/onnela-lab/forest
```
Which will install from GitHub, but the version hasn't changed in 3+ years (Nov 2020 commit): https://github.com/onnela-lab/forest/commit/063558ca67fcab709a336a9840021889d7dd68c5
where it set the (current) forest version) to 0.1.1:
https://github.com/onnela-lab/forest/blob/063558ca67fcab709a336a9840021889d7dd68c5/setup.py#L21
which it is still currently at: https://github.com/onnela-lab/forest/blob/e0bfab42f763e3c76676c5c5085c2b92a7e6e7db/setup.py#L30

Also, is there an intention to release this to PiPy or a repository?

**Comment by hackdna** [comment]
Hi John, thank you for using Forest and raising this issue. The library is under active development and you can get the latest code by using the `develop` branch as you described. We simply neglected bumping the version. We are aiming to have a more regular release schedule and publish the package to PyPI starting in the next couple of months.

**Comment by muschellij2** [comment]
I understand, but it's really hard from a reproducibility perspective, especially as things are dropping/moving around and it makes it hard to publish with this method reliably

**Comment by hackdna** [comment]
Yes, the `develop` branch is meant to support quick feature rollout. This is why we are planning to create tagged releases on the `main` branch on a semi-regular schedule going forward.

In the meantime, you can reference a particular commit during installation for better reproducibility, for example:
```bash
pip install git+https://github.com/onnela-lab/forest@e0bfab42f763e3c76676c5c5085c2b92a7e6e7db
```

**Comment by muschellij2** [comment]
Agreed, but the issue with this is that we had people working on different commit IDs and it's non-trivial for some users to know their commit ID (e.g. `pip freeze`) versus all having version 0.1.1.  My request was a simple version bump here and there to make it easier to track down issues.

**Comment by hydawo** [comment]
Hi @hackdna @biblicabeebli 

Notes from our call on Fri 8/30/24 are here - 

One item JP asked me to add is that we should include the screen-time repo in the official release of Forest 1.0. I'll keep you updated on the changelog progress. Thank you!

**Comment by muschellij2** [comment]
Is https://github.com/onnela-lab/forest/releases/tag/v0.2 on pipy?

**Comment by hackdna** [comment]
Sorry for the delay. The 1.0 release is out now:
https://github.com/onnela-lab/forest/releases/tag/v1.0
https://pypi.org/project/beiwe-forest/
